src/css/Nav.css

```diff
@@ -39,6 +48,12 @@
   font-weight: 400;
   font-size: 1.5em;
   margin: 1%;
+  color: black;
```

Comment:
Could we remove the transform (row 30) on the active button? i.e., transform: none;